scss/footer.scss

```diff
@@ -0,0 +1,29 @@
+footer {
+  align-items: center;
+  background-color: $blue;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 40px 0 10px 0;
+
+  * {
```

Comment:
remember to be generic with the styles you are applying. Apply this in each style you want.